sklearn/linear_model/least_angle.py

```diff
@@ -403,8 +403,11 @@ def lars_path(X, y, Xy=None, Gram=None, max_iter=500,
                 del coef, alpha, prev_alpha, prev_coef
                 # resize the coefs and alphas array
                 add_features = 2 * max(1, (max_features - n_active))
-                coefs = np.resize(coefs, (n_iter + add_features, n_features))
-                alphas = np.resize(alphas, n_iter + add_features)
+                coefs = np.concatenate((coefs,
+                                        np.zeros((add_features, n_features))),
+                                       axis=0)
```

Comment:
can you bench if this is faster:

coefs = np.resize(coefs, (n_iter + add_features, n_features))
coefs[:-add_features, :] = 0.